### DISCUSSION TITLE
Does this preset is working in standard Vue 3 + Vite + Unocss setup

### DISCUSSION THREAD

**Discussion by 0x67** [unocss-applet/unocss-applet#85] [General]
My company use uniapp for few different applications, webview, h5 (desktop), etc. Usually we use this [template](https://github.com/oyjt/uniapp-vue3-template/commits/main/).  

However we also planning to explore Storybook and exporting the components to separate library. Now the question, will this library compatible for cross platforms (Uniapp + Vue3 + Unocss) or [Vitesse](https://github.com/antfu-collective/vitesse) or Nuxt.

In theory I just need the `rpx2rem` conversion.

**Comment by nei1ee** [comment]
This project just adds some presets based on `UnoCSS`, which is cross-platform supported. If you only need `@unocss-applet/preset-rem-rpx` you can just install [`@unocss-applet/preset-rem-rpx`](https://github.com/unocss-applet/unocss-applet/tree/main/packages/preset-rem-rpx) and turn on presets as needed.
